app/api/margatsni/v1/entities/user.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Margatsni
+  module V1
+    module Entities
+      class User < Grape::Entity
+        expose :username
+        expose :email
+        expose :token do |user|
```

Comment:
User entity shouldn't include any tokens. Only user data allowed!

Also, don't forget to provide record id in response